packages/embark-swarm/src/storageProcessesLauncher.js

```diff
@@ -0,0 +1,195 @@
+import { __ } from 'embark-i18n';
```

Comment:
This file is supposed to be used by both Swarm and IPFS, but here you just duplicated it inside both their directories. We should instead put it in a shared directory/package. It can be utils or storage, where you can do `import {storageProcessesLauncher} from '@embark/storage';

Reply:
Agreed.